packages/uniforms/src/types.ts

```diff
@@ -50,6 +55,12 @@ export type GuaranteedProps<Value> = {
   value?: Value;
 };
 
+export type HTMLFieldProps<Value, Element, Extension = {}> = FieldProps<
```

Comment:
I chose the order Value, Extension, Element and gave Element a default because most fields use div as an outer wrapper and thus that argument can often be left out, while most fields have something for the extension. Not to disagree with your choice, just explaining mine :slightly_smiling_face:

Reply:
I've thought about it as well, but I see _some_ value in explicitly setting the `Element`.